tests/write_test.h

```diff
@@ -166,6 +171,31 @@ void write(podio::EventStore& store, WriterT& writer) {
     }
     //-------------------------------
 
+    // ----------------- create a second MC collection -----------------
+    // Can use it to test subset collections that store elements from multiple
+    // collections
+    for (const auto&& mc : mcps) {
+      moreMCs.push_back(mc.clone());
+    }
+
+    // ----------------- add all "odd" mc particles into a subset collection
+    for (auto p : mcps) {
+      if (p.id() % 2) {
+        mcpsRefs.push_back(p);
+      }
+    }
+    // ----------------- add the "even" counterparts from a different collection
+    for (auto p : moreMCs) {
+      if (p.id() % 2 == 0) {
+        mcpsRefs.push_back(p);
+      }
+    }
```

Comment:
> 2. check the subset collection for adding objects from two different collections.

I added this test here that uses original objects from two different collections, and so also shows that the subset collections do not technically have to be subsets of one collection.

Reply:
thanks